### ISSUE TITLE
[Enhancement] Add invisible click handler to charts

### ISSUE THREAD

**Issue by lydemann** [kirbydesign/designsystem#1723]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

Add invisible click handler to charts

## Describe the enhancement

Add invisible click handler to charts as stacked invisible handler, for bar and column charts

## Describe the solution you'd like

Like for the Highcharts charts, an invisible click handler should be added to the charts

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](../CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](../CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
@lydemann what is the use case for this? I am not quite sure I understand what an invisible click handler is, as opposed to the one that you would add via custom options, as described in [interacting with charts](https://cookbook.kirby.design/home/showcase/chart).

**Comment by lydemann** [comment]
It's the same behavior as the deprecated chart, with making it easier to click on a column if it is small by registering the click above using a stacked invisible click handler

**Comment by RasmusKjeldgaard** [comment]
@lydemann Do you have a visualization?

**Comment by lydemann** [comment]
It is hard to visualize something that is invisible. You click above the column and the column gets selected, that's it.

**Comment by RasmusKjeldgaard** [comment]
Ah okay, I think I can help you with a visualization then. If I click where the dot is, it should select the bar, right?
[Chartjs invisible bar.pdf](https://github.com/kirbydesign/designsystem/files/7075628/Chartjs.invisible.bar.pdf)

**Comment by RasmusKjeldgaard** [comment]
@henrikvoetmand and I agree that it should be default behavior, and thus implemented in Kirby, and not via customOptions in a project.

**Comment by MadsBuchmann** [comment]
Still relevant - would be a nice addition to the chart.

**Comment by RasmusKjeldgaard** [comment]
We are closing this issue, as a lot of work has happened on charts since it was opened. If this is still needed, please open a new issue with updated information.
